python/pdo/contract/state.py

```diff
@@ -248,19 +252,12 @@ def get_current_state_hash(ledger_config, contract_id) :
     # --------------------------------------------------
     @classmethod
     def get_from_ledger(cls, ledger_config, contract_id, current_state_hash = None) :
```

Comment:
please consider using the `deprecated` property in `python/pdo/common/utility.py`
or perhaps remove the function if it is no longer useful

Reply:
This is tracked via issue 263. throwing exception for now since the call won't work with CCF (the call will be used if running multiple clients on multiples nodes).